src/fitnesse/wikitext/shared/ContentsItemBuilder.java

```diff
@@ -77,8 +78,27 @@ else if (hasOption("-H", Names.HELP_INSTEAD_OF_TITLE_TOC)) {
         return listItem;
     }
 
+    private int getTotalTestPagesInASuite(SourcePage page, int counter) {
+      if (page.hasProperty(PageType.TEST.toString())) {
+        return ++counter;
+      }
+      if (page.hasProperty(PageType.SUITE.toString())) {
+        Iterator<SourcePage> pages = page.getChildren().iterator();
+        while (pages.hasNext()) {
+          SourcePage sourcePage = pages.next();
+          counter = getTotalTestPagesInASuite(sourcePage, counter);
+        }
+      }
+      return counter;
+    }
+
     private String buildBody(SourcePage page) {
         String itemText = page.getName();
+        //Will show count of test pages under this suite
+        if (hasOption("-c", Names.TEST_PAGE_COUNT_TOC)) {
+          if (page.hasProperty(PageType.SUITE.toString()))
+            itemText += "(Test pages:" + getTotalTestPagesInASuite(page, 0) + ")";
```

Comment:
```suggestion
            itemText += " (Test pages: " + getTotalTestPagesInASuite(page, 0) + ")";
```

A few spaces will make this easier to read.
I actually wonder: is the `Test pages:` needed, why not only have the count between parentheses?

Reply:
Thanks. I changed it.